include/experimental/__p0009_bits/compressed_pair.hpp

```diff
@@ -28,8 +28,8 @@ namespace detail {
 // For no unique address emulation, this is the case taken when neither are empty.
 // For real `[[no_unique_address]]`, this case is always taken.
 template <class _T1, class _T2, class _Enable = void> struct __compressed_pair {
-  _MDSPAN_NO_UNIQUE_ADDRESS _T1 __t1_val;
-  _MDSPAN_NO_UNIQUE_ADDRESS _T2 __t2_val;
+  _MDSPAN_NO_UNIQUE_ADDRESS _T1 __t1_val{};
+  _MDSPAN_NO_UNIQUE_ADDRESS _T2 __t2_val{};
```

Comment:
Default initialization of nested members of mdspan, otherwise it would be not initialized in defaulted constructor.

Reply:
If neither type is empty in this specialization, why do you need `_MDSPAN_NO_UNIQUE_ADDRESS` at all?

Reply:
I think the comment above is the answer: if we do use real "unique address" attribute with compilers which support it, this is the only specialization of compressed pair which exists, i.e. we also use it for empty members.